prototype2/multigrid/mgmesytec/MG24GeometryTest.cpp

```diff
@@ -1,15 +1,15 @@
 /** Copyright (C) 2016, 2017 European Spallation Source ERIC */
 
 #include <libs/include/TSCTimer.h>
-#include <multigrid/mgmesytec/MG24Detector.h>
+#include <multigrid/mgmesytec/MG24Geometry.h>
 #include <test/TestBase.h>
 
 class MG24DetectorTest : public TestBase {};
 
 /** Test cases below */
 
 TEST_F(MG24DetectorTest, IsWireIsGrid) {
-  MG24Detector mgdet;
+  MG25Geometry mgdet;
```

Comment:
MG25 ?

Reply:
Yeah, that's wrong, but actually I have removed this class even from building. This is all sort of dead code. With what we did at Sequoia with multi-bus setup, this old class kind of becomes more work to maintain than it's probably worth. Should we just excise it completely?

Reply:
We still have detector data from the smaller detector prototype. I think this is where that code was used. Is the new code compatible or can we no longer parse and create events correctly for  the old data?

Reply:
But do we really care about that data anymore? I just emailed Anton about this. The impression I got was that this was never going to be used.  As the detectors go through prototypes, we refine our code as well, but maintaining support for the older ones is maybe not good use of our time?

Reply:
I can see the argument, but we need to be 100% sure before chucking it out. Anyway this is the data that we use for the integration test and the change will break that.

So you should probably get a wireshark with data taken from the recent detector and create a configuration file and demonstrate this. Then we need to give that to Afonso and push the data to the correct locations/servers....

Reply:
Ok, I have a response from Anton. He does indeed still intend to use MG24, so I will fix this class, but probably only after vacation.